x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/AbstractAggregationTestCase.java

```diff
@@ -435,16 +440,23 @@ private IntBlock makeGroupsVector(int groupStart, int groupEnd, int rowCount) {
      */
     private void processPageGrouping(GroupingAggregator aggregator, Page inputPage, int groupCount) {
         var groupSliceSize = 1;
+        var allValuesNull = IntStream.range(0, inputPage.getBlockCount())
```

Comment:
Looks like this is what the `LongBlockHash` `SeenGroupIds` is doing (more or less), and was needed to fix the tests for Max/Min.

It is now working, and I think it makes sense. But will need a double check here

Reply:
`SeenGroupIds` is a kind of to handle the cases where you start off receiving no `null` values and then start receiving `null` values later. That case is fun and maybe we should get there some day. We have tests for it over in the compute engine. Maybe these tests can get there soon and we can replace them. Not sure. But this is fine for now.